eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md

```diff
@@ -0,0 +1,210 @@
+# EiffelSourceChangeCreatedEvent
+The EiffelSourceChangeCreatedEvent declares that a change to sources has been made, but not yet submitted (see [EiffelSourceChangeSubmittedEvent](./EiffelSourceChangeSubmittedEvent.md)). This can be used to represent a change done on a private branch, undergoing review or made in a forked repository. Unlike EiffelSourceChangeSubmittedEvent, EiffelSourceChangeCreatedEvent _describes the change_ in terms of who authored it and which issues it addressed.
+
+Where changes are integrated (or "submitted") directly on a shared branch of interest (e.g. "master", "dev" or "mainline") both EiffelSourceChangeCreatedEvent and EiffelSourceChangeSubmittedEvent are sent together.
+
+## Data Members
+### data.author
+__Type:__ Object  
+__Required:__ No  
+__Description:__ The author of the change.
+
+#### data.author.name
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the author.
+
+#### data.author.email
+__Type:__ String  
+__Required:__ No  
+__Description:__ The email address of the author.
+
+#### data.author.id
+__Type:__ String  
+__Required:__ No  
+__Description:__ Any identity, username or alias of the author.
+
+#### data.author.group
+__Type:__ String  
+__Required:__ No  
+__Description:__ Any group or organization to which the contributor belongs.
+
+### data.change
+__Type:__ Object  
+__Required:__ No  
+__Description:__ A summary of the change.
+
+#### data.change.insertions
+__Type:__ Integer  
+__Required:__ No  
+__Description:__ The number of inserted lines in the change.
+
+#### data.change.deletions
+__Type:__ Integer  
+__Required:__ No  
+__Description:__ The number of deleted lines in the change.
+
+#### data.change.files
+__Type:__ String  
+__Required:__ No  
+__Description:__ A URI to a list of files changed, on JSON String array format.
+
+### data.issues
+__Type:__ Object[]  
+__Required:__ No  
+__Description:__ A list of issues addressed by the change.
+
+#### data.issues.type
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ BUG, IMPROVEMENT, FEATURE, WORK_ITEM, REQUIREMENT, OTHER  
```

Comment:
I disagree. The event says that something happened: sources were changed. Why did this happen? Because of a work item, saying this had to be done. At the time you look at the event the work item may be open or closed or anything in between, but that's besides the point. You still want to be able to trace changes you made (and thereby product deltas) to work items.